middleware/jwt-session.js

```diff
@@ -22,7 +23,11 @@ module.exports = {
 // as as a JSON response body (for clients that may not be able to
 // work with cookies).
 async function login(req, res) {
-  const token = await sign({ email: req.user.email, id: req.user.id });
+  const token = await sign({
+    email: req.user.email,
+    id: req.user.id,
+    sub: `${req.user.role}` || "data_entry",
```

Comment:
the role(s) set on the user by `account-service.js` gets signed onto the jwt subject key here, and then later decoded and matched on validation